eventservice/api/controllers/events.py

```diff
@@ -655,7 +655,7 @@ def event_operation_permission_check(req_data=None, event_id=None):
                 if not check_group_event_admin_access(req_data, group_memberships):
                     abort(401)
     elif req_data:
-        if not req_data.get('createdByGroupId'):
+        if not req_data.get('createdByGroupId') and not event.get('groupIds'):
```

Comment:
@bingzhang, the new addition should be `and not req_data.get('groupIds')`. The event object will be None in this case as it's a POST.

Reply:
fixed